src/test/scala/sangria/validation/rules/OverlappingFieldsCanBeMergedFastSpec.scala

```diff
@@ -0,0 +1,800 @@
+package sangria.validation.rules
+
+import org.scalatest.WordSpec
+import sangria.schema._
+import sangria.util.{Pos, ValidationSupport}
+
+class OverlappingFieldsCanBeMergedFastSpec extends WordSpec with ValidationSupport {
+
+  override val defaultRule = Some(new OverlappingFieldsCanBeMergedFast)
```

Comment:
General question:
if I try this tests with the current implementation (`override val defaultRule = Some(new OverlappingFieldsCanBeMerged)`, I have some errors.
Is it expected?

Reply:
While the rules do implement the same validation, they differ in how they report the errors, so they cannot use the same set of tests. See my answer to your general questions below for more on this.